pkgs/development/compilers/gcc-arm-embedded/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, bzip2, patchelf, glibc, gcc, fetchurl, version, releaseType, sha256, ncurses
```

Comment:
This file is not used in this partial revert (it was used only pre-gcc6 before the removing commit).

I'd think either drop it, or re-add gcc<6 to all-packages?